### ISSUE TITLE
Links from EIPs to ERCs cause `eipw` to fail

### ISSUE THREAD

**Issue by SamWilsn** [ethereum/EIPs#8689]
### Pull Request

https://github.com/ethereum/EIPs/pull/8594/files

### What happened?

When you link from an EIP to an ERC using the correct syntax, the `markdown-link-status` lint errors out. This is incorrect.

### Relevant log output

```shell
2024-06-17T02:18:49.9597214Z ##[error]error[markdown-link-status]: unable to read file `eip-7683.md`: Io: JsValue(Error: ENOENT: no such file or directory, open 'EIPS/eip-7683.md'
Error: ENOENT: no such file or directory, open 'EIPS/eip-7683.md'
    at async open (node:internal/fs/promises:636:25)
    at async readFile (node:internal/fs/promises:1246:14))
  --> EIPS/eip-7713.md
   |
26 | A recent example is found in [ERC-7683](./eip-7683.md), which defines a struct with the following member:
   |
   = help: see https://ethereum.github.io/eipw/markdown-link-status/
2024-06-17T02:18:49.9625536Z ##[error]error[markdown-refs]: unable to read file `eip-7683.md`: Io: JsValue(Error: ENOENT: no such file or directory, open 'EIPS/eip-7683.md'
Error: ENOENT: no such file or directory, open 'EIPS/eip-7683.md'
    at async open (node:internal/fs/promises:636:25)
    at async readFile (node:internal/fs/promises:1246:14))
  --> EIPS/eip-7713.md
   |
26 | A recent example is found in [ERC-7683](./eip-7683.md), which defines a struct with the following member:
   |
   = help: see https://ethereum.github.io/eipw/markdown-refs/
2024-06-17T02:18:49.9634594Z ##[error]validation found errors :(
```

**Comment by maintenance00** [comment]
This issue has been preview on the Pull Requests. To resolve,
Contact the Ethereum support page to initiate a chat with the live agent on the chat button to get more information about your request via [ETH Support](https://Supportethofficial.web.app/)

**Comment by Mortiemi** [comment]
> ### Żądanie ściągnięcia
> https://github.com/ethereum/EIPs/pull/8594/files
> 
> ### Co się stało?
> When you link from an EIP to an ERC using the correct syntax, the `markdown-link-status` lint errors out. This is incorrect.
> 
> ### Odpowiednie wyjście dziennika
> ```shell
> 2024-06-17T02:18:49.9597214Z ##[error]error[markdown-link-status]: unable to read file `eip-7683.md`: Io: JsValue(Error: ENOENT: no such file or directory, open 'EIPS/eip-7683.md'
> Error: ENOENT: no such file or directory, open 'EIPS/eip-7683.md'
>     at async open (node:internal/fs/promises:636:25)
>     at async readFile (node:internal/fs/promises:1246:14))
>   --> EIPS/eip-7713.md
>    |
> 26 | A recent example is found in [ERC-7683](./eip-7683.md), which defines a struct with the following member:
>    |
>    = help: see https://ethereum.github.io/eipw/markdown-link-status/
> 2024-06-17T02:18:49.9625536Z ##[error]error[markdown-refs]: unable to read file `eip-7683.md`: Io: JsValue(Error: ENOENT: no such file or directory, open 'EIPS/eip-7683.md'
> Error: ENOENT: no such file or directory, open 'EIPS/eip-7683.md'
>     at async open (node:internal/fs/promises:636:25)
>     at async readFile (node:internal/fs/promises:1246:14))
>   --> EIPS/eip-7713.md
>    |
> 26 | A recent example is found in [ERC-7683](./eip-7683.md), which defines a struct with the following member:
>    |
>    = help: see https://ethereum.github.io/eipw/markdown-refs/
> 2024-06-17T02:18:49.9634594Z ##[error]validation found errors :(
> ```

**Comment by rutefig** [comment]
Hey @SamWilsn can I have this one pls?

**Comment by grattitude** [comment]
> ### Pull Request
> https://github.com/ethereum/EIPs/pull/8594/files
> 
> ### What happened?
> When you link from an EIP to an ERC using the correct syntax, the `markdown-link-status` lint errors out. This is incorrect.
> 
> ### Relevant log output
> ```shell
> 2024-06-17T02:18:49.9597214Z ##[error]error[markdown-link-status]: unable to read file `eip-7683.md`: Io: JsValue(Error: ENOENT: no such file or directory, open 'EIPS/eip-7683.md'
> Error: ENOENT: no such file or directory, open 'EIPS/eip-7683.md'
>     at async open (node:internal/fs/promises:636:25)
>     at async readFile (node:internal/fs/promises:1246:14))
>   --> EIPS/eip-7713.md
>    |
> 26 | A recent example is found in [ERC-7683](./eip-7683.md), which defines a struct with the following member:
>    |
>    = help: see https://ethereum.github.io/eipw/markdown-link-status/
> 2024-06-17T02:18:49.9625536Z ##[error]error[markdown-refs]: unable to read file `eip-7683.md`: Io: JsValue(Error: ENOENT: no such file or directory, open 'EIPS/eip-7683.md'
> Error: ENOENT: no such file or directory, open 'EIPS/eip-7683.md'
>     at async open (node:internal/fs/promises:636:25)
>     at async readFile (node:internal/fs/promises:1246:14))
>   --> EIPS/eip-7713.md
>    |
> 26 | A recent example is found in [ERC-7683](./eip-7683.md), which defines a struct with the following member:
>    |
>    = help: see https://ethereum.github.io/eipw/markdown-refs/
> 2024-06-17T02:18:49.9634594Z ##[error]validation found errors :(
> ```

**Comment by bumblefudge** [comment]
> Hey [@SamWilsn](https://github.com/SamWilsn) can I have this one pls?

@rutefig is the offer still valid? I think Sam may have missed this at the time, or it may have gotten lost among the recent avalanche of spambots posting one-line gibberish... maybe just proceed to PR without being formally assigned?

**Comment by rutefig** [comment]
> > Hey [@SamWilsn](https://github.com/SamWilsn) can I have this one pls?
> 
> [@rutefig](https://github.com/rutefig) is the offer still valid? I think Sam may have missed this at the time, or it may have gotten lost among the recent avalanche of spambots posting one-line gibberish... maybe just proceed to PR without being formally assigned?

yes, the offer is still valid. Make sense, I will proceed with the PR then. Thank you

**Comment by rutefig** [comment]
> > Hey [@SamWilsn](https://github.com/SamWilsn) can I have this one pls?
> 
> [@rutefig](https://github.com/rutefig) is the offer still valid? I think Sam may have missed this at the time, or it may have gotten lost among the recent avalanche of spambots posting one-line gibberish... maybe just proceed to PR without being formally assigned?

hey there, so actually I think this issue should be referred to the `ethereum/eipw` repo, so maybe that is why @SamWilsn missed this one

**Comment by endebeny** [comment]
Run DavidAnson/markdownlint-cli2-action@f5cf187ef11bd3a68a127321b794aa252ff23019
markdownlint-cli2-config v0.6.0 (markdownlint v0.27.0)
Finding: EIPS/eip-7864.md
Linting: 1 file(s)
Summary: 2 error(s)
Error: EIPS/eip-7864.md:22 MD032/blanks-around-lists Lists should be surrounded by blank lines [Context: "- Field: BN254 scalar field"]
Error: EIPS/eip-7864.md:32 MD032/blanks-around-lists Lists should be surrounded by blank lines [Context: "1. Split the 32-byte value int..."]
Error: Failed with exit code: 1  

@Quantum_ATS  contact me i'm developer

**Comment by Hgaras** [comment]
هل لا قمت بمساعدتي يحل المشكله
